src/main/java/controller/LottoGameController.java

```diff
@@ -42,13 +45,30 @@ private LottoMoney getLottoMoney() {
 		}
 	}
 
+	private Lottos buyLottos(LottoMoney lottoMoney) {
+		int manualLottoCount = lottoGameView.getManualLottoCount();
+		List<Lotto> manualLottos = getManualLottos(manualLottoCount);
+
+		LottoMoney remain = lottoMoney.calculateRemainAfterBuy(manualLottoCount);
+		int autoLottoCount = remain.calculateLottoCount();
+		List<Lotto> autoLottos = lottoGenerator.generateLottos(autoLottoCount);
+
+		lottoGameView.printPurchasedLottoCount(manualLottoCount, autoLottoCount);
+		return new Lottos(Stream.concat(manualLottos.stream(), autoLottos.stream())
+			.collect(Collectors.toList()));
```

Comment:
아래와 같이 manualLottos와 autoLottos를 각각의 일급 컬렉션(= Lottos)로 생성하고 이 두 컬렉션을 합쳐서 새로운 일급 컬렉션을 생성하면 어떨까요? 😃

```suggestion
		Lottos manualLottos = getManualLottos(manualLottoCount);

		LottoMoney remain = lottoMoney.calculateRemainAfterBuy(manualLottoCount);
		int autoLottoCount = remain.calculateLottoCount();
		Lottos autoLottos = lottoGenerator.generateLottos(autoLottoCount);

		lottoGameView.printPurchasedLottoCount(manualLottoCount, autoLottoCount);
		return manualLottos.plus(autoLottos);
```

Reply:
`Lottos#concat` 추가 했습니다!
```java
public Lottos concat(Lottos lottos) {
	return new Lottos(Stream.concat(this.lottos.stream(), 
			lottos.getLottos().stream())
		.collect(Collectors.toList()));
}
```
그리고 로또 데이터를 담고있는 객체들을 `domain.lotto`로 옮겼어요 (로또 관련 도메인 객체들이 많아짐)